src/test/java/org/openrewrite/java/migrate/UpgradeToJava6Test.java

```diff
@@ -0,0 +1,137 @@
+                  // TODO Auto-generated method stub
+                  return null;
+                }
+                                  
+                public void setLogWriter(PrintWriter out) throws SQLException {
+                  // TODO Auto-generated method stub
+                }
+                                  
+                public void setLoginTimeout(int seconds) throws SQLException {
+                  // TODO Auto-generated method stub
+                }
+                                  
+                public int getLoginTimeout() throws SQLException {
+                  // TODO Auto-generated method stub
+                  return 0;
+                }
+              }
+              """,
+            """
+              import java.io.PrintWriter;
+              import java.sql.Connection;
+              import java.sql.SQLException;
+                               
+              import javax.sql.DataSource;
+                               
+              public class JRE6WrapperDataSource implements DataSource {
+                               
+                public Connection getConnection() throws SQLException {
+                  // TODO Auto-generated method stub
+                  return null;
+                }
+                               
+                public Connection getConnection(String username, String password)
+                    throws SQLException {
+                  // TODO Auto-generated method stub
+                  return null;
+                }
+                               
+                public PrintWriter getLogWriter() throws SQLException {
+                  // TODO Auto-generated method stub
+                  return null;
+                }
+                               
+                public void setLogWriter(PrintWriter out) throws SQLException {
+                  // TODO Auto-generated method stub
+                }
+                
+                public void setLoginTimeout(int seconds) throws SQLException {
+                  // TODO Auto-generated method stub
+                }
+                               
+                public int getLoginTimeout() throws SQLException {
+                  // TODO Auto-generated method stub
+                  return 0;
+                }
+                            
+                  public boolean isWrapperFor(Class<?> iface) throws java.sql.SQLException {
+                      return iface != null && iface.isAssignableFrom(this.getClass());
+                  }
+               
+                  public <T> T unwrap(Class<T> iface) throws java.sql.SQLException {
+                      try {
+                          if (isWrapperFor(iface)) {
+                              return (T) this;
+                          }
+                          throw DbException.getInvalidValueException("iface", iface);
+                      } catch (Exception e) {
+                          throw logAndConvert(e);
```

Comment:
This `logAndConvert` isn't defined anywhere; we likely need to insert something here that's safe to insert in any context. I don't know the specifics of `unwrap` well enough to know what exactly, but possible it's to wrap the exception into an `SQLException`.